edi/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 ACSONE
+# @author: Simone Orsi <simahawk@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Base EDI Backend",
+    "summary": """Base module to define EDI backends""",
+    "version": "13.0.1.0.0",
+    "development_status": "Alpha",
+    "license": "AGPL-3",
+    "author": "ACSONE,Odoo Community Association (OCA)",
+    "maintainers": ["simahawk"],
+    "depends": ["base_edi", "connector"],
```

Comment:
Thanks for this module.
Just a question here. Do you think that connector is mandatory in this module or only component/queue_job ? Maybe connector in an additional module ?

Reply:
At least component is required (with the changes we are working on), and probably connector will be required.

Reply:
yeah, for now we can stick to component only indeed. Fixed.